web/themes/custom/itkdev/itkdev_project_theme/templates/views-exposed-form.html.twig

```diff
@@ -0,0 +1,23 @@
+{#
+/**
+ * @file
+ * Default theme implementation of a views exposed form.
+ *
+ * Available variables:
+ * - form: A render element representing the form.
+ *
+ * @see template_preprocess_views_exposed_form()
+ *
+ * @ingroup themeable
+ */
+#}
+{% if q is not empty %}
+  {#
+    This ensures that, if clean URLs are off, the 'q' is added first,
+    as a hidden form element, so that it shows up first in the POST URL.
```

Comment:
Are you sure? If you `POST` you don't add anything to the URL. Do you mean `GET`? You should always use `GET` to get stuff (Drupal breaks this rule in multiple places).

Reply:
I am not sure. I copy pasted this solely to add the `grid` classes to `<form...`. should I change the comment to `GET`?

Reply:
Can't (better) exposed filters add the css class names? And make sure that you `GET`?